src/module/Alias/src/Alias/AliasManager.php

```diff
@@ -327,8 +327,13 @@ protected function slugify($text)
         $slugified = [];
 
         foreach (explode('/', $text) as $token) {
-            $text        = $slugify->filter($token);
-            $text        = $shortify->filter($text);
+            if (empty($token)) {
+                continue;
+            }
+
+            $text        = $shortify->filter($token);
```

Comment:
You can move this in front of the foreach loop: `$text = $shortify->filter($text);`